app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/covid_19_claim_check.scala.html

```diff
@@ -0,0 +1,62 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.views.html.templates.principal_main_template
+@import uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.config.AppConfig
+@import uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.models.YesNo
+@import uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.forms.Covid19ClaimCheckForm._
+@import uk.gov.hmrc.play.views.html.helpers.form
+@import uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.views.html.helpers._
+
+@(covid19ClaimCheckForm: Form[YesNo], postAction: Call)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@principal_main_template(title = messages("covid_19_claim_check.title")) {
+@summaryErrorHelper(covid19ClaimCheckForm)
+<h1>@messages("covid_19_claim_check.title")</h1>
+
+<p>@messages("covid_19_claim_check.join_pilot")</p>
+
+<ul class="list list-bullet">
+    <li>@messages("covid_19_claim_check.join_pilot_point1.part1") <a id="supportScheme" href="@appConfig.covid19SupportSchemeUrl" target="_blank">@messages("covid_19_claim_check.join_pilot_point1.link_text")</a> @messages("covid_19_claim_check.join_pilot_point1.part2")</li>
```

Comment:
this is quite a long line, I understand what you are doing, but the messages outside of the link could be combined like:
```suggestion
    <li>@Html(messages(
            "covid_19_claim_check.join_pilot_point1", <a id="supportScheme" href={appConfig.covid19SupportSchemeUrl} target="_blank">{messages("covid_19_claim_check.join_pilot_point1.link_text")}</a>
        ))</li>
```
Then the covid_19_claim_check.join_pilot_point1 would be "have claimed through the {0} for sole traders"
So passing the link through to the message as a parameter
I don't mind either way though, if you want to keep the way it currently is, just set this to resolved

Reply:
The problem with your recommended approach is that we have to hard code URL in html file rather than from appconfig i.e., we can't use @appConfig.url. I don't mind either ways but considering above issue I'll suggest to leave it as it is.

Reply:
The above suggestion works and allows the use of appConfig